terraform/monitoring.tf

```diff
@@ -1,19 +1,31 @@
 
 module "prometheus_all" {
-  source = "git::https://github.com/DFE-Digital/cf-monitoring.git//prometheus_all"
+  source = "git::https://github.com/DFE-Digital/cf-monitoring.git//prometheus_all?ref=435-move-paas-app-alerting-to-cf-monitoring"
```

Comment:
Shouldn't this be main branch?.

Reply:
Yes. It will be before the merge. I am testing in local on feature branch.